cpp/cmake_modules/BuildUtils.cmake

```diff
@@ -406,7 +406,8 @@ function(ADD_ARROW_LIB LIB_NAME)
                                                                    "${_lib_install_name}")
     endif()
 
-    install(TARGETS ${LIB_NAME}_shared ${INSTALL_IS_OPTIONAL}
+    install(TARGETS ${LIB_NAME}_shared ${ARG_SHARED_INSTALL_EXTRA_TARGETS}
+                    ${INSTALL_IS_OPTIONAL}
```

Comment:
This change and the one below are also unnecessary right?

Reply:
Removed.